framework/build.gradle

```diff
@@ -149,6 +154,26 @@ jacocoTestReport {
     }
     getExecutionData().setFrom(fileTree('../framework/build/jacoco').include("**.exec"))
 }
+ */
+
+offlinsCoverage {
+    jacocoVersion = '0.8.8' // Optional. By default `0.8.8`
+
+    reports {
+        html.enabled.set true // Optional. By default `true`
+        html.location.set project.file('build/reports/jacoco/test/html') // Optional. By default `build/reports/jacoco/html`
+
+        xml.enabled.set true // Optional. By default `false`
+        xml.location.set project.file('build/reports/jacoco/test/jacocoTestReport.xml')
+        // Optional. By default `build/reports/jacoco/coverageReport.xml`
+
+        csv.enabled.set true // Optional. By default `false`
+        csv.location.set project.file('build/reports/jacoco/test/csvCoverage.csv')
+        // Optional. By default `build/reports/jacoco/coverageReport.csv`
+    }
+}
+
+
```

Comment:
Can the Jacoco coverage of each module still be output locally after this change?

Reply:
Yes, and the path is still the same

Reply:
Yes, and the path is still the same

Reply:
Yes, and the path is still the same